include/core/rheological_model.h

```diff
@@ -72,6 +72,15 @@ class RheologicalModel : public PhysicalPropertyModel
     return 1.0;
   }
 
+  /**
+   * @brief Returns the viscosity scale. This is used to get the viscosity scale in the problem when it's not constant. This is useful for weak boundary conditions.
+   */
+  virtual double
+  get_viscosity_scale() const
+  {
+    return 1.0;
```

Comment:
Why does this return 1.0 ? Should this return the viscosity and, if you are non-newtonian, something like eta_0, or eta_inf, or the average of both? This 1 is random. I think this function should be a pure virtual function....